editor.planx.uk/src/@planx/components/Pay/Public/FeeBreakdown/utils.ts

```diff
@@ -5,24 +5,52 @@ import { FeeBreakdown, PassportFeeFields } from "./types";
 export const toNumber = (input: number | [number]) =>
   Array.isArray(input) ? input[0] : input;
 
+/**
+ * Convert a Passport value to an actual boolean
+ */
+const toBoolean = (val: ["true" | "false"]) => val[0] === "true";
```

Comment:
Let's make a proper lib of passport helpers in planx-core or similar one of these days :dizzy:

Reply:
Totally!